dependency/health_service_test.go

```diff
@@ -20,45 +22,45 @@ func TestNewHealthServiceQuery(t *testing.T) {
 		name string
 		i    string
 		exp  *HealthServiceQuery
-		err  bool
+		err  error
 	}
 	cases := tenancyHelper.GenerateTenancyTests(func(tenancy *test.Tenancy) []interface{} {
 		return []interface{}{
 			testCase{
 				tenancyHelper.AppendTenancyInfo("empty", tenancy),
 				"",
 				nil,
-				true,
+				fmt.Errorf(`health.service: invalid format: ""`),
 			},
 			testCase{
 				tenancyHelper.AppendTenancyInfo("dc_only", tenancy),
 				"@dc1",
 				nil,
-				true,
+				fmt.Errorf(`health.service: invalid format: "@dc1"`),
 			},
 			testCase{
 				tenancyHelper.AppendTenancyInfo("near_only", tenancy),
 				"~near",
 				nil,
-				true,
+				fmt.Errorf(`health.service: invalid format: "~near"`),
 			},
 			testCase{
 				tenancyHelper.AppendTenancyInfo("tag_only", tenancy),
 				"tag.",
 				nil,
-				true,
+				fmt.Errorf(`health.service: invalid format: "tag."`),
 			},
 			testCase{
 				tenancyHelper.AppendTenancyInfo("query_only", tenancy),
 				fmt.Sprintf("?ns=%s", tenancy.Namespace),
 				nil,
-				true,
+				fmt.Errorf(`health.service: invalid format: "?ns=%s"`, tenancy.Namespace),
 			},
 			testCase{
 				tenancyHelper.AppendTenancyInfo("invalid query param (unsupported key)", tenancy),
 				"name?unsupported=test",
 				nil,
-				true,
+				fmt.Errorf(`health.service: invalid query parameter key "unsupported" in query "unsupported=test": supported keys: ns,peer,partition`),
```

Comment:
shouldn't the list of supported keys include `sameness-group`? Maybe we forgot to update something somewhere?

Reply:
that is a good observation. let me check consul.

Reply:
this validation was in consul-template.  fixed this in 2896a992ebd4710ccb9e67c6962ca4f1207b7181